cli/popper/utils.py

```diff
@@ -342,3 +342,39 @@ def find_recursive_wfile():
                 wfile = os.path.abspath(wfile)
                 wfile_list.append(wfile)
     return wfile_list
+
+
+def parse(url):
```

Comment:
this can also take care of extracting the version, and a `path` that would represent anything between the `<action>` and the `@` sign.